web-client/public/js/container.js

```diff
@@ -50,80 +51,78 @@ export const container = function () {
+    //     $(selector).addClass("glyphicon-ok");
+    // };
 
-    $("#boundBoxS").on("click", function () {
-        $("#boundBoxS").prop("checked", true);
-        small();
-    });
+    // var small = function () {
+    //     setSizeIndicator("#viewport-size-s span");
+    // };
 
-    $("#viewport-size-m").on("click", function () {
-        $("#boundBoxM").prop("checked", true).trigger("click");
-        medium();
-    });
+    // var medium = function () {
+    //     setSizeIndicator("#viewport-size-m span");
+    // };
 
-    $("#boundBoxM").on("click", function () {
-        $("#viewport-size-m").prop("checked", true);
-        medium();
-    });
+    // var large = function () {
+    //     setSizeIndicator("#viewport-size-l span");
+    // };
 
-    $("#viewport-size-l").on("click", function () {
-        $("#boundBoxL").prop("checked", true).trigger("click");
-        large();
-    });
+    // var fit = function () {
+    //     setSizeIndicator("#viewport-size-fit span");
+    // };
 
-    $("#boundBoxL").on("click", function () {
-        $("#viewport-size-l").prop("checked", true);
-        large();
-    });
-
-    $("#viewport-size-fit").on("click", function () {
-        $("#boundBoxFit").prop("checked", true).trigger("click");
-        fit();
-    });
+    requestWindowDimensions();
 
-    $("#boundBoxFit").on("click", function () {
-        $("#viewport-size-fit").prop("checked", true);
-        fit();
-    });
+    if (pageWidth < MEDIUM_PAGE_WIDTH) {
+        grnState.viewportSize = "containerS";
+    } else if (pageWidth > MEDIUM_PAGE_WIDTH && pageWidth < LARGE_PAGE_WIDTH) {
+        grnState.viewportSize = "containerM";
+    } else {
+        grnState.viewportSize = "containerL";
+    }
+    updateApp(grnState);
+
+    // $("#viewport-size-s").on("click", function () {
+    //     $("#boundBoxS").prop("checked", true).trigger("click");
+    //     small();
+    // });
+
+    // $("#boundBoxS").on("click", function () {
+    //     $("#boundBoxS").prop("checked", true);
+    //     small();
+    // });
+
+    // $("#viewport-size-m").on("click", function () {
+    //     $("#boundBoxM").prop("checked", true).trigger("click");
+    //     medium();
+    // });
+
+    // $("#boundBoxM").on("click", function () {
+    //     $("#viewport-size-m").prop("checked", true);
+    //     medium();
+    // });
+
+    // $("#viewport-size-l").on("click", function () {
+    //     $("#boundBoxL").prop("checked", true).trigger("click");
+    //     large();
+    // });
+
+    // $("#boundBoxL").on("click", function () {
+    //     $("#viewport-size-l").prop("checked", true);
+    //     large();
+    // });
+
+    // $("#viewport-size-fit").on("click", function () {
+    //     $("#boundBoxFit").prop("checked", true).trigger("click");
+    //     fit();
+    // });
+
+    // $("#boundBoxFit").on("click", function () {
+    //     $("#viewport-size-fit").prop("checked", true);
+    //     fit();
+    // });
```

Comment:
If the fix is verified, probably safe to remove completely in order to avoid confusion.